src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -57,7 +57,7 @@ def convert_tf_weight_name_to_pt_weight_name(
           transposed with regards to each other
     """
     if name_scope is not None:
-        if not tf_name.startswith(name_scope):
+        if not tf_name.startswith(name_scope) and "final_logits_bias" not in tf_name:
```

Comment:
Why the check on "final_logits_bias" here?

Reply:
No good reason, unfortunately! BART and derived models put the `final_logits_bias` weights outside the main model name scope, and I'm not entirely sure why (but changing it now might invalidate old user checkpoints)